tests/flows/native_token_bridge.go

```diff
@@ -0,0 +1,383 @@
+
+		nativeTokenDestinationAbi, err := nativetokendestination.NativeTokenDestinationMetaData.GetAbi()
+		Expect(err).Should(BeNil())
+		utils.DeployContract(
+			ctx,
+			NativeTokenDestinationByteCodeFile,
+			deployerPK,
+			subnetB,
+			nativeTokenDestinationAbi,
+			teleporterContractAddress,
+			subnetA.BlockchainID,
+			bridgeContractAddress,
+			initialReserveImbalance,
+		)
+
+		log.Info("Finished deploying Bridge contracts")
+	}
+
+	// Create abi objects to call the contract with
+	nativeTokenDestination, err := nativetokendestination.NewNativeTokenDestination(
+		bridgeContractAddress,
+		subnetB.WSClient,
+	)
+	Expect(err).Should(BeNil())
+	nativeTokenSource, err := nativetokensource.NewNativeTokenSource(
+		bridgeContractAddress,
+		subnetA.WSClient,
+	)
+	Expect(err).Should(BeNil())
+
+	// Helper function
+	sendTokensToSource := func(valueToSend *big.Int, fromKey *ecdsa.PrivateKey, toAddress common.Address) *types.Receipt {
+		transactor, err := bind.NewKeyedTransactorWithChainID(fromKey, subnetB.EVMChainID)
+		Expect(err).Should(BeNil())
+		transactor.Value = valueToSend
+
+		tx, err := nativeTokenDestination.TransferToSource(
+			transactor,
+			toAddress,
+			emptyDestFeeInfo,
+			[]common.Address{},
+		)
+		Expect(err).Should(BeNil())
+		log.Info(
+			"Sent TransferToSource transaction on destination chain",
+			"sourceBlockchainID",
+			subnetA.BlockchainID,
+			"txHash",
+			tx.Hash().Hex(),
+		)
+
+		destChainReceipt := utils.WaitForTransactionSuccess(ctx, tx.Hash(), subnetB)
+
+		transferEvent, err := utils.GetEventFromLogs(
+			destChainReceipt.Logs,
+			nativeTokenDestination.ParseTransferToSource,
+		)
+		Expect(err).Should(BeNil())
+		utils.ExpectBigEqual(transferEvent.Amount, valueToSend)
+
+		receipt := network.RelayMessage(ctx, destChainReceipt, subnetB, subnetA, true)
+
+		return receipt
+	}
+
+	// Helper function
+	sendTokensToDestination := func(
+		valueToSend *big.Int,
+		fromKey *ecdsa.PrivateKey,
+		toAddress common.Address,
+	) *types.Receipt {
+		transactor, err := bind.NewKeyedTransactorWithChainID(fromKey, subnetA.EVMChainID)
+		Expect(err).Should(BeNil())
+		transactor.Value = valueToSend
+
+		tx, err := nativeTokenSource.TransferToDestination(
+			transactor,
+			toAddress,
+			emptySourceFeeInfo,
+			[]common.Address{},
+		)
+		Expect(err).Should(BeNil())
+		log.Info(
+			"Sent TransferToDestination transaction on source chain",
+			"destinationBlockchainID",
+			subnetB.BlockchainID,
+			"txHash",
+			tx.Hash().Hex(),
+		)
+
+		sourceChainReceipt := utils.WaitForTransactionSuccess(ctx, tx.Hash(), subnetA)
+
+		transferEvent, err := utils.GetEventFromLogs(
+			sourceChainReceipt.Logs,
+			nativeTokenSource.ParseTransferToDestination,
+		)
+		Expect(err).Should(BeNil())
+		utils.ExpectBigEqual(transferEvent.Amount, valueToSend)
+
+		receipt := network.RelayMessage(ctx, sourceChainReceipt, subnetA, subnetB, true)
+
+		return receipt
+	}
+
+	{ // Transfer some tokens A -> B
+		// Check starting balance is 0
```

Comment:
It might just be the Github UI, but the formatting here looks off. Can you start the comment on the line below the opening curly brace for readability on Github? Same for the `{}` blocks below.